docs-data/overrides.json

```diff
@@ -948,6 +948,11 @@
                 "description": "An optional plain text user NKey seed to use along with the corresponding user JWT."
               }
             ]
+          },
+          {
+            "name": "tls_handshake_first",
+            "description": "Whether to perform the initial TLS handshake before sending the NATS INFO protocol message. This can be required when connecting to some NATS servers that expect TLS to be established immediately after connection, before any protocol negotiation.",
```

Comment:
"can be required"? Seems like we should be able to say why/when it is required, and when it isn't.
Should this be:
Whether to perform the initial TLS handshake before sending the NATS INFO protocol message. This is required when connecting to some NATS servers that expect TLS to be established immediately after connection, before any protocol negotiation.